apps/glusterfs/db_operations.go

```diff
@@ -226,8 +226,11 @@ func DbDump(jsonfile string, dbfile string) error {
 	if err != nil {
 		return fmt.Errorf("Could not construct dump from DB: %v", err.Error())
 	}
+	enc := json.NewEncoder(fp)
+	//set indentation for json data
```

Comment:
The comment here is a bit redundant as the next line is clearly setting the indent. I'd either reword the comment to provide more detail about why the indentation is being set OR just drop the comment.

Reply:
@phlogistonjohn, exactly! Avoid redundant comments!
@Madhu-1, please either remove the comment or explain what's going on.... :-)